src/lib/Server/Controller/BinaryContent.php

```diff
@@ -37,17 +38,16 @@ public function __construct(
      * Returns data about the image variation $variationIdentifier of image field $fieldId.
      * Will generate the alias if it hasn't been generated yet.
      *
-     * @param mixed  $imageId A custom ID that identifies the image field.
+     * @param mixed $imageId A custom ID that identifies the image field.
      *                        Until v6.9, the format is {contentId}-{fieldId}.
      *                        since v6.9, the format is {contentId}-{fieldId}-{versionNumber}.
      *                        If the version number isn't specified, the default one is used.
-     * @param string $variationIdentifier
      *
      * @throws \Ibexa\Contracts\Rest\Exceptions\NotFoundException
      *
      * @return \Ibexa\Contracts\Core\Variation\Values\Variation
      */
-    public function getImageVariation($imageId, $variationIdentifier)
+    public function getImageVariation(mixed $imageId, string $variationIdentifier): Variation|CachedValue
```

Comment:
`$imageId` is definitely a string. Is there any usage which calls it with something else?